### ISSUE TITLE
IE11 tiles not loading when mapbox-gl used with Browserify

### ISSUE THREAD

**Issue by rustyb** [mapbox/mapbox-gl-js#2635]
Hi Folks - ran into a weird issue with IE11 and including mapbox-gl in a bundle.js created with Browserify. 

The map initialises (as in the controls and bg colour show) but it doesn't load the tiles nor display any errors. Please see screenshot below:

![virtualbox_virtual_test_30_05_2016_19_11_56](https://cloud.githubusercontent.com/assets/53086/15655120/7a6ee290-269a-11e6-9c95-d4066792986e.png)

[Gist Source](https://gist.github.com/rustyb/e6b1cbd7f6232beb0a8de80d339f0ea7) 

[RawGIT example](https://rawgit.com/rustyb/e6b1cbd7f6232beb0a8de80d339f0ea7/raw/f337f4047925f4821cb3befff3d766371fa18be7/index.html#3.42/52.41/19.56)

Using mapbox-gl v18.0.

Interestingly if you add mapbox-gl in a script tag in the header of the html:
```javascript 
<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.18.0/mapbox-gl.js'></script>
```
and remove ```var mapboxgl = require('mapbox-gl');``` the tiles actually load.

Any ideas on how one might avoid this?

**Comment by roelvanhintum** [comment]
Did you get one of the examples running on IE11?
https://www.mapbox.com/mapbox-gl-js/examples/

**Comment by rustyb** [comment]
Yup the examples I tried seemed to all run fine.

Except those that loaded a geojson source.

**Comment by roelvanhintum** [comment]
I can't even load the default one, which doesn't load anything except the map itself.
https://www.mapbox.com/mapbox-gl-js/examples/
I created an issue for that example #2677

**Comment by lucaswoj** [comment]
Fixed in #2504! Release coming soon.
